tests/utils/create-agent.js

```diff
@@ -0,0 +1,17 @@
+const getSessionToken = require('./get-session-token')
```

Comment:
Using a session cookie as a way to authenticate to the API is different from using a bearer token in the authentication headers, and ties the API authentication mechanism to the website behavior rather than keeping it independent. I'm not sure if there's an easy way around this at the moment.